pandas/_libs/lib.pyx

```diff
@@ -150,11 +150,11 @@ def memory_usage_of_objects(arr: object[:]) -> int64_t:
 
     Does not include the actual bytes of the pointers
     """
-    i: Py_ssize_t
-    n: Py_ssize_t
-    size: int64_t
+    cdef:
+        Py_ssize_t i
+        Py_ssize_t n
+        int64_t size = 0
```

Comment:
this is no longer supported?  IIRC i did this as part of a (doomed) effort to move the code towards being valid python, partially in the hopes that we could adapt linting tools someday.